app/controllers/AdminController.php

```diff
@@ -176,7 +176,7 @@ public function updateGatewayRequest(){
                 $username = Session::get("username");
                 $email = Config::get('pga_config.portal')['admin-emails'];
                 $user_profile = Keycloak::getUserProfile($username);
-                EmailUtilities::mailToUser($user_profile["firstname"], $user_profile["lastname"], $email, Input::get("gateway_id"));
+                EmailUtilities::mailToUser($user_profile["firstname"], $user_profile["lastname"], $user_profile["email"], Input::get("gateway_id"));
```

Comment:
This code is called when the admin updates the gateway request, right? If so, this will send an email to the admin because it is getting the email address of the admin's (currently logged in user) user profile.

Instead it should fetch the Gateway and use the emailAddress field there.